scripts/build.ts

```diff
@@ -350,48 +423,10 @@ export default async function(
   }
 
   logger.info('Setting versions...');
+  await _setVersions(sortedPackages, argv, logger.createChild('versions'));
 
-  const versionLogger = logger.createChild('versions');
-  for (const packageName of sortedPackages) {
-    versionLogger.info(packageName);
-    const pkg = packages[packageName];
-    const packageJsonPath = path.join(pkg.dist, 'package.json');
-    const packageJson = pkg.packageJson;
-    const version = pkg.version;
-
-    if (version) {
-      packageJson['version'] = version;
-    } else {
-      versionLogger.error('No version found... Only updating dependencies.');
-    }
-
-    for (const depName of Object.keys(packages)) {
-      const v = packages[depName].version;
-      for (const depKey of ['dependencies', 'peerDependencies', 'devDependencies']) {
-        const obj = packageJson[depKey] as JsonObject | null;
-        if (obj && obj[depName]) {
-          if (argv.local) {
-            obj[depName] = packages[depName].tar;
-          } else if (argv.snapshot) {
-            const pkg = packages[depName];
-            if (!pkg.snapshotRepo) {
-              versionLogger.error(
-                `Package ${JSON.stringify(depName)} is not published as a snapshot. `
-                + `Fixing to current version ${v}.`,
-              );
-              obj[depName] = v;
-            } else {
-              obj[depName] = `github:${pkg.snapshotRepo}#${pkg.snapshotHash}`;
-            }
-          } else if ((obj[depName] as string).match(/\b0\.0\.0\b/)) {
-            obj[depName] = (obj[depName] as string).replace(/\b0\.0\.0\b/, v);
-          }
-        }
-      }
-    }
-
-    fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2) + '\n');
-  }
+  logger.info('Setting packageGroups...');
+  await _setPackageGroups(sortedPackages, logger.createChild('packageGroups'));
```

Comment:
Won't this add package groups to all the dev kit packages each with all the other dev kit packages included?

Reply:
I'll change it a bit.

Reply:
Done, I added a check instead that build will add package group packages if they're in the package.json already, and added all packages that I thought were okay.